Fix installing to use the new assembly names--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,7 @@
 
 install: Geckofx
 	install -d $(DESTDIR)
-	install GeckoFxTest/bin/x86/Debug_Linux/geckofx-core-${FF_MAJOR}.dll $(DESTDIR)/geckofx-core-${FF_MAJOR}.dll
-	install GeckoFxTest/bin/x86/Debug_Linux/geckofx-core-${FF_MAJOR}.dll.config $(DESTDIR)/geckofx-core-${FF_MAJOR}.dll.config
-	install GeckoFxTest/bin/x86/Debug_Linux/Geckofx-Winforms-${FF_MAJOR}.dll $(DESTDIR)/Geckofx-Winforms-${FF_MAJOR}.dll
+	install GeckoFxTest/bin/x86/Debug_Linux/Geckofx-Core.dll $(DESTDIR)/Geckofx-Core.dll
+	install GeckoFxTest/bin/x86/Debug_Linux/Geckofx-Core.dll.config $(DESTDIR)/Geckofx-Core.dll.config
+	install GeckoFxTest/bin/x86/Debug_Linux/Geckofx-Winforms.dll $(DESTDIR)/Geckofx-Winforms.dll
 	install Geckofx-Core/Linux/geckofix.so $(DESTDIR)/geckofix.so
